validation/td-validation.ttl

```diff
@@ -256,7 +256,7 @@
     <code>readOnly</code> and <code>writeOnly</code> members, among
     others.
 </p>
-<p><code>PropertyAffordance</code> <a>Class</a> is a <a>Subclass</a> of the class <code>InteractionAffordance</code>. 
+<p><code>PropertyAffordance</code> is a <a>Subclass</a> of the <code>InteractionAffordance</code> <a>Class</a>. 
```

Comment:
of the <code>InteractionAffordance</code> <a>Class</a> and the <code>DataSchema</code> <a>Class</a>.